src/map/status.c

```diff
@@ -8080,15 +8136,34 @@ static int status_change_start_sub(struct block_list *src, struct block_list *bl
 				val4 = INVALID_TIMER; //Kaahi Timer.
 				break;
 			case SC_BLESSING:
-				if ((!undead_flag && st->race!=RC_DEMON) || bl->type == BL_PC)
-					val2 = val1;
-				else
-					val2 = 0; //0 -> Half stat.
+			{
+				int bonus = 0;
+				struct map_session_data *srcsd = BL_CAST(BL_PC, src);
+				if (skill_id == AB_CLEMENTIA && srcsd != NULL)
+					bonus += srcsd->status.job_level / 10;
+
+				if ((!undead_flag && st->race!=RC_DEMON) || bl->type == BL_PC) {
```

Comment:
Missing spaces around !=

Reply:
thanks! fixed it and rebased

And don't worry about it being in draft, as I put in the PR description, it is ready, but moving this PR alone may not be a good idea since kRO moved the 2nd jobs and transclasses at the same time.

I am just keeping it in draft until I can PR the other parts, in separate PRs so we don't have a single hundred commits PR :) but reviewing it now is totally fine